src/components/shared/container/container.jsx

```diff
@@ -0,0 +1,24 @@
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Container = ({ className: additionalClassName, children, ...otherProps }) => {
+  const className = clsx('relative mx-auto px-safe', additionalClassName);
```

Comment:
Container should not have relative class by default, it should be optional. Also what are the advantages to use it as a separate component instead of just using classnames? It feels like it would be easier to have for this just a dedicated class with apply.

Reply:
1. `Container` should always have relative class by default since all absolute content should be positioned relatively to container, that's what `Container` is for.
2. Component is a better choice since:
  2.1. We have always used it like this
  2.2. It's easily scalable
  2.3. It's easier to read the code when you see Container rather than `div className="container"`

Reply:
1. There could be scenarios when it would negatively effect on the development. For most of the cases relative is good, but I don't think we need to put it just in case and instead using it where we actually need it.

2. 
2.1. No. 
2.2 In some cases yes, in some cases no. Example, we need to add framer motion, with this component you won't be able to add here "motion.div".  You won't be able to use react intersection observer with current implementation either.
2.3 Subjective :) For me it feels the same, the only difference is that with Container component I receive additional "magic" classes, that are not visible for me.

So having single class with apply, looks for me like a better scalable option, since it has nothing to do with js, has no additional logic despite giving us necessary styles.